infrastructure/cdn-in-a-box/traffic_ops_integration_test/Dockerfile

```diff
@@ -82,6 +85,7 @@ COPY ./infrastructure/cdn-in-a-box/traffic_ops_integration_test/config.sh /opt/i
 COPY ./traffic_ops/testing/api/v1/tc-fixtures.json /opt/integration/app/tc-fixtures-v1.json
 COPY ./traffic_ops/testing/api/v2/tc-fixtures.json /opt/integration/app/tc-fixtures-v2.json
 COPY ./traffic_ops/testing/api/v3/tc-fixtures.json /opt/integration/app/tc-fixtures-v3.json
+COPY ./traffic_ops/testing/api/v4/tc-fixtures.json /opt/integration/app/tc-fixtures-v4.json
```

Comment:
I'm pretty sure these four copies can be a single cache layer

Reply:
It doesn't just copy the files, it gives each of them a new name. I'm not sure how to make that a single cache layer.

Reply:
Oh, so it does.